examples/qclustering/oracle.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+
+from qibo.gates import Unitary
+from qibo.models import Circuit
+
+
+def f(x, threshold):
+    return x < threshold
```

Comment:
This one line function has only one call, I suggest removing it.

Reply:
It is there more for someone to see they can modify the f(x) how they prefer.